web/wp-content/themes/dctx-v2/inc/queries.php

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-rgnotw.php, but found phpcs-scan-rgNotW.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).